tests/appsec/iast_packages/test_packages.py

```diff
@@ -109,56 +141,309 @@ def install_latest(self):
+        "charset-normalizer",
+        "3.3.2",
+        "my-bytes-string",
+        "my-bytes-string",
+        "",
+        import_name="charset_normalizer",
+        import_module_to_validate="charset_normalizer.api",
+    ),
+    PackageForTesting("click", "8.1.7", "", "", "", test_e2e=False, import_module_to_validate="click.core"),
+    PackageForTesting(
+        "cryptography",
+        "42.0.7",
+        "This is a secret message.",
+        "This is a secret message.",
+        "",
+        import_module_to_validate="cryptography.fernet",
+    ),
+    PackageForTesting("distlib", "0.3.8", "", "", "", test_e2e=False, import_module_to_validate="distlib.util"),
+    PackageForTesting(
+        "exceptiongroup", "1.2.1", "", "", "", test_e2e=False, import_module_to_validate="exceptiongroup._formatting"
+    ),
+    PackageForTesting("filelock", "3.14.0", "", "", "", test_e2e=False, import_module_to_validate="filelock._api"),
+    PackageForTesting("flask", "2.3.3", "", "", "", test_e2e=False, import_module_to_validate="flask.app"),
+    PackageForTesting("fsspec", "2024.5.0", "", "/", ""),
+    PackageForTesting(
+        "google-api-core",
+        "2.19.0",
+        "",
+        "",
+        "",
+        import_name="google",
+        import_module_to_validate="google.auth.iam",
     ),
     PackageForTesting(
         "google-api-python-client",
         "2.111.0",
         "",
         "",
         "",
-        extras=[("google-auth-oauthlib", "1.2.0"), ("google-auth-httplib2", "0.2.0")],
+        extras=[("google-auth-oauthlib", "1.2.0"), ("google-auth-httplib2", "0.2.0"), ("cryptography", "42.0.7")],
         import_name="googleapiclient",
+        import_module_to_validate="googleapiclient.discovery",
+    ),
+    PackageForTesting(
+        "idna",
+        "3.6",
+        "xn--eckwd4c7c.xn--zckzah",
+        "ドメイン.テスト",
+        "xn--eckwd4c7c.xn--zckzah",
+        import_module_to_validate="idna.codec",
+    ),
+    PackageForTesting(
+        "importlib-resources",
+        "6.4.0",
+        "",
+        "",
+        "",
+        test_e2e=False,
+        import_name="importlib_resources",
+        skip_python_version=[(3, 8)],
+        import_module_to_validate="importlib_resources.readers",
+    ),
+    PackageForTesting("isodate", "0.6.1", "", "", "", test_e2e=False, import_module_to_validate="isodate.duration"),
+    PackageForTesting(
+        "itsdangerous", "2.2.0", "", "", "", test_e2e=False, import_module_to_validate="itsdangerous.serializer"
+    ),
+    PackageForTesting("jinja2", "3.1.4", "", "", "", test_e2e=False, import_module_to_validate="jinja2.compiler"),
+    PackageForTesting("jmespath", "1.0.1", "", "Seattle", "", import_module_to_validate="jmespath.functions"),
+    # jsonschema fails for Python 3.8
+    #        except KeyError:
+    # >           raise exceptions.NoSuchResource(ref=uri) from None
+    # E           referencing.exceptions.NoSuchResource: 'http://json-schema.org/draft-03/schema#'
```

Comment:
Should we add a task to look into it?

Reply:
Not really since it also fails the unpatched version. Probably a package issue.